src/dali/dali_main.py

```diff
@@ -182,7 +183,7 @@ def _setup_argument_parser() -> ArgumentParser:
         "-c",
         "--use-cache",
         action="store_true",
-        help="Cache input plugin data load (developers only)",
+        help="Cache input plugin data load",
```

Comment:
The caching infrastructure for plugin is meant to speed up development, not for the final user. This is mainly because it lacks the ability to check if new data is available, so the user might get old results without realizing it (i.e. only what is in the cache even though new transactions may be available). I would leave the dev only notice.

Reply:
This was completely unintentional. No matter how careful I am, I always manage to mess something up with git. How can I just overwrite my copy of dali_main.py with the one from main? Basically, disregard any changes I made. 

Is this the correct way:

`git checkout main src/dali_main.py`

Reply:
No worries: that's what reviews are for :-) 

To revert changes I typically do:
```
git checkout src/dali/dali_main.py
```

Still, let me know if you want me to review your commits at any time or you prefer me to wait for your go-ahead. Either way is good with me, thanks!